sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.bulkInsert.e2e-spec.ts

```diff
@@ -405,13 +405,15 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-bulkInsert", () =>
               sabcProgramCode: csvProgramSABCCodeSBC9,
               startDate: "2023-09-02",
               endDate: "2024-08-12",
-              errors: ["Mandatory fees must be not greater than 999999."],
+              errors: [
+                `Mandatory fees must be not greater than ${MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE}.`,
+              ],
               infos: [],
               warnings: [
                 {
                   typeCode:
                     OfferingValidationWarnings.OfferingCostExceedMaximum,
-                  message: "Mandatory fees must be not greater than 100000.",
+                  message: `Mandatory fees must be not greater than 100000.`,
```

Comment:
I think we could either use double quotes here or use `MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE` in the message.

Reply:
I am changing to double quotes. MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE is not applicable to this message as this is for maximum offering cost to be auto approved.